### PR TITLE
Adapt design from SimpleRow

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/Sailfish/Silica/ListItem.qml [hunk 1]
```diff
@@ -9,8 +9,23 @@ SimpleRow {
 
     height: contentHeight
 
-    // HACK: Display arrow on the bottom right
-    text: " "
-    detailText: " "
+    // Do not show right angle in iOS
```
Comment (micuintus) [inline qml/Sailfish/Silica/ListItem.qml]: "Angle", really? I know, the v-play guys use that term, as well.
But I think "arrow" would be the better term ;).

File: qml/pages/VenueList.qml [hunk 2]
```diff
@@ -155,6 +155,7 @@ Page {
                     leftMargin: Theme.horizontalPageMargin
                     rightMargin: Theme.horizontalPageMargin
 
+                    topMargin: Theme.paddingSmall
```
Comment (micuintus) [inline qml/pages/VenueList.qml]: Imo that worsens the look of the Sailfish app a little bit.
Old:
old
old2

New:
new

We probably need a our own Theme API that we implement for both Sailfish and v-play (as opposed to only implementing the Sailfish API for v-play).

But for now, this is okay.

File: qml/Sailfish/Silica/ListItem.qml (hunk 1 above)
Comment (jmastr) [inline qml/Sailfish/Silica/ListItem.qml]: This comes from FontAwesome: They also have "arrow right", which is something different to "angle right".

File: qml/pages/VenueList.qml (hunk 2 above)
Comment (jmastr) [inline qml/pages/VenueList.qml]: I don't think that it worsens it, but why is the distance base-aligned to the street? That's a bug right?

Comment (jmastr) [inline qml/pages/VenueList.qml]: virtualbox_sailfishos emulator_16_02_2017_08_22_27

Look at the distance. That's how `development` currently (0b4dd4c) looks like.

**Pull request merged**